Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/root/.virtualenvs/portal/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/root/.virtualenvs/portal/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/root/.virtualenvs/portal/local/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/root/.virtualenvs/portal/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/root/.virtualenvs/portal/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/root/portal/sigaram/portaladmin/models.py", line 60, in <module>
    class Admininfo(models.Model):
  File "/root/portal/sigaram/portaladmin/models.py", line 63, in Admininfo
    username = models.ForeignKey(Logininfo, to_field='username')
NameError: name 'Logininfo' is not defined
